.gitlab/functional_test/regression_detector.yml

```diff
@@ -83,13 +83,13 @@ single-machine-performance-regression_detector:
             --submission-metadata submission_metadata
     # Wait for job to complete.
     - RUST_LOG="${RUST_LOG}" ./smp --team-id ${SMP_AGENT_TEAM_ID} --api-base ${SMP_API} --aws-named-profile ${AWS_NAMED_PROFILE}
-            job status
+            job status --use-curta
```

Comment:
I believe this also needs a consignor flag

Reply:
Corrected in [fcabd46](https://github.com/DataDog/datadog-agent/pull/18566/commits/fcabd466bbaedff010a2e879170cb37e1891df7b).